src/components/Header/NavBar.js

```diff
@@ -18,12 +19,20 @@ const StyledLink = Styled(Link)`
 `;
 
 
-export default function NavBar() {
+export default function NavBar({ searchPath }) {
```

Comment:
The reason why we don't need it as a prop here is that the Navbar link will always point to the same URL. Since it doesn't change we don't need to have a dynamic prop